internal/generator/vector/output/otlp/otlp_test.go

```diff
@@ -46,10 +65,13 @@ var _ = Describe("Generate vector config", func() {
 		if visit != nil {
 			visit(&outputSpec)
 		}
+		var conf []framework.Element
 		if tune {
 			adapter = *fake.NewOutput(outputSpec, secret, framework.NoOptions)
+			conf = New(helpers.MakeOutputID(outputSpec.Name), outputSpec, []string{"pipeline_my_pipeline_viaq_0"}, secret, adapter, op)
+		} else {
+			conf = New(helpers.MakeOutputID(outputSpec.Name), outputSpec, []string{"pipeline_my_pipeline_viaq_0"}, secret, nil, op)
```

Comment:
This `else` isn't needed because `adapter` is `nil` unless it's set by `tune == true`.

Reply:
if `tune` was set `true` one time for other tests after`adapter` will initialize